lib/gears/async_popen.lua

```diff
@@ -0,0 +1,59 @@
+---------------------------------------------------------------------------
+-- @release @AWESOME_VERSION@
+-- @module gears.async_popen
+---------------------------------------------------------------------------
+
+local setmetatable = setmetatable
+local util = require("awful.util")
+
+local async_popen = { mt = {} }
+
+-- Maps temporary filenames to callbacks
+async_popen.callbacks = {}
+
+--- Run a command asynchronously.
+-- @param command The command to run
+-- @param callback The function to be called when the command exits. The output of the command will be passed to it.
```

Comment:
Please use `@tparam` and `@treturn` here, i.e. `@tparam string command ...`.

Also, please wrap lines at max. 80 chars, indenting them in the documentation with two spaces.